### PR TITLE
Bugfix: fix build of xforms

### PR DESCRIPTION
**Pull request by stefanfechter**
This additional patch fixes the build of the now unmaintained library xforms.

### CODE REVIEW

File: var/spack/repos/builtin/packages/xforms/package.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Xforms(AutotoolsPackage):
     depends_on("libx11", type="link")
     depends_on("libxpm", type="link")
     depends_on("jpeg", type="link")
+    patch("xformsPatch.patch")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/xforms/package.py:23]: Do you know if the patch applies to all listed versions or for some version on?

See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#patches.

**Comment by tldahlgren** [comment]
@ketsubouchi As the original contributor, do you want to comment on this PR?

**Comment by tldahlgren** [comment]
@spackbot run pipeline

File: var/spack/repos/builtin/packages/xforms/package.py (hunk 1 above)
Comment (stefanfechter) [reply var/spack/repos/builtin/packages/xforms/package.py:23]: That patch is needed for the last version of xforms, 1.2.4 . Not sure about the other versions though.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/xforms/package.py:23]: Do you want to add the `when="@1.2.4"` argument to the patch then?

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**